src/plugins/DragDrop/index.js

```diff
@@ -111,32 +104,34 @@ module.exports = class DragDrop extends Plugin {
   }
 
   onBrowseClick (ev) {
-    const input = this.el.querySelector('.uppy-DragDrop-input')
-    input.click()
+    this.input.click()
   }
 
   render (state) {
-    return html`
-      <div class="Uppy UppyTheme--default uppy-DragDrop-container ${this.isDragDropSupported ? 'is-dragdrop-supported' : ''}"
-           style="width: ${this.opts.width}; height: ${this.opts.height};">
-        <form class="uppy-DragDrop-inner" onsubmit=${(ev) => ev.preventDefault()}>
-          <svg class="UppyIcon uppy-DragDrop-arrow" aria-hidden="true" width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+    return (
+      <div class={`Uppy UppyTheme--default uppy-DragDrop-container ${this.isDragDropSupported ? 'is-dragdrop-supported' : ''}`}
+           style={`width: ${this.opts.width}; height: ${this.opts.height};`}
```

Comment:
do you have any opinions for or against using style objects here?

Reply:
For style objects, because this is messy :) Will update, thanks.